app/helpers/creators_helper.rb

```diff
@@ -0,0 +1,3 @@
+module CreatorsHelper
```

Comment:
If you don't have time, I understand, but I like to delete these unused auto-generated files.